internal/exec/resolvable/resolvable.go

```diff
@@ -241,9 +265,12 @@ func buildDirectivePackers(s *types.Schema, visitors map[string]directives.Direc
 			continue
 		}
 
-		if _, ok = v.(directives.ResolverInterceptor); !ok {
+		switch v.(type) {
+		default:
```

Comment:
Went with default first here since it’s a break early not a success flow, but will change it back to default last (which I had originally before I reordered)